dynamic-simulation/dynamic-simulation-tool/src/main/java/com/powsybl/dynamicsimulation/tool/DynamicSimulationTool.java

```diff
@@ -134,8 +143,14 @@ public void run(CommandLine line, ToolRunningContext context) throws Exception {
 
         DynamicSimulation.Runner runner = DynamicSimulation.find();
 
-        Path dydFile = context.getFileSystem().getPath(line.getOptionValue(DYNAMIC_MODEL_FILE));
-        DynamicModelsSupplier dydSupplier = createMappingSupplier(dydFile, runner.getName());
+        Path dydFile = context.getFileSystem().getPath(line.getOptionValue(DYNAMIC_MODELS_FILE));
+        DynamicModelsSupplier dynamicModelsSupplier = createMappingSupplier(dydFile, runner.getName());
```

Comment:
Hum, I'm also thinking createMappingSupplier should be renamed into createDynamicModelsSupplier to be consistent

Reply:
Done. Method createEventSupplier renamed to createEventModelsSupplier too.